cmd/v/help/where.txt

```diff
@@ -23,19 +24,18 @@ symbol_name can be:
 
 params:
   -mod  [mod_name]      Restrict to search recursively only within of the given 
-                        module, if not provided search in entire v scope.
+                        module, if not provided search in entire v scope,
+                        RECOMMENDATION: use -mod main
```

Comment:
265 / 5.000
Resultados de traducción
well, the purpose of that recommendation is to search the main module (which would be the base directory of the project) recursively. To avoid searching the entire scope of v, if not necessary. But there's certainly no reason for it to be in all caps.